data/successors/successor news.txt

```diff
@@ -0,0 +1,277 @@
+# Copyright (c) 2024 by Daeridanii
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+
+
+news "high houses scion"
+	to show
+		has "language: Successor"
+	location
+		attributes successor
+		attributes wealthy
+		not attributes uninhabited
+	name
+		word
+			"High Houses Scion"
+			"Child of the High Houses"
+			"Scion of the High Houses"
+	message
+		word
+			`"`
+		word
+			"I hope thou has't enjoyed thy time amongst our people."
+			"The alien! Doth thou have news from far in this endless sky?"
+			"Fair traveler, I've heard of thee. How must thy voyage weigh upon thy mind."
+			"Welcome to our space, honored alien. A pleasure 'tis to meet thee."
+		word
+			`"`
+
+
+news "high houses official"
+	to show
+		has "language: Successor"
+	location
+		attributes successor
+		not attributes crime uninhabited
+	name
+		word
+			"High Houses official"
+			"High Houses officer"
+	message
+		word
+			`"`
+		word
+			"The Houses welcome thee as 'guest' among our people; "
+			"Please remain in concord with our people; "
+			"Informed, I am, of thy recent arrival; "
+			"Can I provide assistance, honored traveler? "
+			"Our noble house does greet thee full of joy; "
+		word
+			"I am at thy service if 'tis needed."
+			"Today's been rather busy, please speak quickly."
+		word
+			`"`
+	message
+		word
+			`"`
+		word
+			"Please submit to scans if 'tis requested"
+			"Taxes on transactions are collected"
+			"Transactions in this area are surveilled"
+			"If thou see'st something suspect, report't promptly"
+		word
+			"; port rules and regulations should be posted."
+			" in accordance with our standard port procedure."
+		word
+			`"`
+
+
+news "successor student"
+	to show
+		has "language: Successor"
+	location
+		attributes successor
+		attributes education wealthy
+		not attributes uninhabited
+	name
+		word
+			"Successor student"
+	message
+		word
+			`"`
+		word
+			"Exams, I think, shall never know their end,"
+			"Hath I forgot my geographic terms?"
+			"Hath I forgot my chemical reagents?"
+			"Hyperspace Mechanics is a bore,"
+			"Predecessors His'try is a drag,"
+			"Plasma Flow Dynamics is a bore,"
+			"Particle Kinetics is a bore,"
```

Comment:
The first thing that came to me reading these was, it would be neat if the adjective used to describe the class was a pun related to the class itself (like Plasma Flow Dynamics being torpid or something like that, although it's also good as is.